tf_quant_finance/black_scholes/vanilla_prices_test.py

```diff
@@ -134,6 +134,31 @@ def test_price_vol_and_expiry_scaling(self):
             forwards=forwards))
     self.assertArrayNear(base_prices, scaled_prices, 1e-10)
 
```

Comment:
Please check both float32 and float64 via parametrized. Here and elsewhere. See, e.g., 
https://github.com/google/tf-quant-finance/blob/514236ce735ee2d6bdd9fb9981ce79da4dcaa1cd/tf_quant_finance/black_scholes/crr_binomial_tree_test.py#L30